crates/sage/src/scoring.rs

```diff
@@ -7,6 +8,12 @@ use serde::Serialize;
 use std::ops::AddAssign;
 use std::sync::atomic::{AtomicUsize, Ordering};
 
+#[derive(Copy, Clone, Debug)]
```

Comment:
Add `Serialize, Deserialize` to auto derives.

Reply:
Added `Serialize, Deserialize`.